packages/landingpage/translations/locales/de/getting-started.json

```diff
@@ -1,6 +1,9 @@
 {
   "sandbox": {
     "title": "Suchen Sie nach weiteren Ressourcen?",
-    "description": "Sehen Sie sich gerne auch unsere Beispielprojekte an"
+    "description": "Sehen Sie sich gerne auch unsere Beispielprojekte an",
+    "goto": "Zum Sandbox-Projekt",
+    "alt": "Sandbox-Projekt für",
+    "aria-label": "Öffnen Sie das Sandbox-Projekt für"
```

Comment:
As of my knowledge cutoff in July 2023 I don't have information as to what is the current strategy, but back then we decided (@janivo and I) that we were going to drop internationalization support for the getting started guides due to the lack of a real demand for german-written guides in the developer community (and the organizational overhead on top of that). This is achieved with the completion of [PR: Remove german guides](https://github.com/inovex/elements/pull/973).

Reply:
Oh I didn't know that, thanks for letting me know 👍